src/codelabs/null-safety.md

```diff
@@ -0,0 +1,357 @@
+---
+title: "Null safety"
```

Comment:
Let's have a different name from the main null safety page (also, we don't need quotation marks here):

```suggestion
title: Null safety codelab
```